data/src/main/java/de/ph1b/audiobook/data/repo/internals/migrations/Migration44to45.kt

```diff
@@ -0,0 +1,9 @@
+package de.ph1b.audiobook.data.repo.internals.migrations
+
+import androidx.sqlite.db.SupportSQLiteDatabase
+
+class Migration44to45 : IncrementalMigration(44) {
```

Comment:
Instead of adding another migration, edit Migration43to44 and add that functionality there.

The last release is on 43 so we don't need another migration.
https://github.com/PaulWoitaschek/Voice/blob/v3.6.3/data/src/main/java/de/ph1b/audiobook/data/repo/internals/InternalDb.kt#L35

Reply:
👍

Reply:
Hint: Navigate to the AppDb_Impl and exactly copy-paste the create table statement room created.

Reply:
Is this for fixing the tests?

Reply:
In the previous migration, all room tables were created from scratch.
It's not only for fixing the tests but for having the correct schema.